docs/command/query.soy

```diff
@@ -321,14 +321,14 @@ the repository and the target <code>//foo:bar</code>.
 </p>
 
 <p>
-This operator is meant to be used together with the <code>--dot</code>
+This operator is best used together with the <code>--dot</code>
 {sp}parameter to generate an image. Sample usage:
 </p>
 
-<p><pre>
+<p><pre class="prettyprint lang-py">
 {literal}
-buck query "allpaths(//foo:bar, //foo/bar/lib:baz)" --dot > result.dot
-dot -Tpng result.dot -o image.png
+$ buck query "allpaths(//foo:bar, //foo/bar/lib:baz)" --dot > result.dot
+$ dot -Tpng result.dot -o image.png
```

Comment:
You're right, it's inconsistent. Should I change this in all 'buck query' calls in this page?